README.md

```diff
@@ -43,7 +48,35 @@ something because there is no value for an env var it was relying on.
 | `SPLUNK_HEC_TOKEN`    | [HTTP Event Collector token](http://dev.splunk.com/view/event-collector/SP-CAAAE7C)    |                          | In `production` |
 | `SPLUNK_HEC_ENDPOINT` | [HTTP Event Collector endpoint](http://dev.splunk.com/view/event-collector/SP-CAAAE7H) |                          | In `production` |
 | `LOG_LEVEL`           | [bunyan log level](https://github.com/trentm/node-bunyan#levels)                       | Depends on `NODE_ENV`    |                 |
-| `DB_ID`               | Name of documentDB                                                                     | services                 |                 |
-| `DB_COLLECTION_ID`    | Name of documentDB collection                                                          | services                 |                 |
-| `DB_ENDPOINT`         | Endpoint for documentDB. Available in the [Azure Portal](https://portal.azure.com), more help [here](https://docs.microsoft.com/en-us/azure/documentdb/documentdb-nodejs-get-started#a-idconfigastep-3-set-your-apps-configurations) | https://connecting-to-services.documents.azure.com:443/ |                 |
-| `DB_PRIMARY_KEY`      | Primary Read-Only Key to access documentDB. Available in the [Azure Portal](https://portal.azure.com), more info [here](https://docs.microsoft.com/en-us/azure/documentdb/documentdb-nodejs-get-started#a-idconfigastep-3-set-your-apps-configurations) | confidential  | Yes             |
+| `MONGO_DB`              | Name of the database in the Mongo image                                              | services                 |                 |
+| `MONGODB_COLLECTION`    | Name of collection in the Mongo image                                                | pharmacies               |                 |
+| `MONGODB_HOST`          | Name of MongoDB host                                                                 | mongo                    |                 |
+| `MONGODB_PORT`          | The port used by MongoDB                                                             | 27017                    |                 |
+
+## Running the application
+
+Before running the application for the first time, copy the `.env-sample` into `.env` and replace the values in `.env`
```

Comment:
This can be removed when using default value variables in the `docker-compose` files.

It might be worth saying something about if you want to have the default environment variables values overridden adding them into a `.env` will do that. Especially useful for the `SNYK_TOKEN`